rllib/core/optim/tests/test_rl_optimizer_tf.py

```diff
@@ -5,19 +5,13 @@
 import unittest
 
 import ray
```

Comment:
You can ignore everything that is under optim. Since these tests are removed from CI anyway.